### PR TITLE
Feat/challenge functions 1

### PR DESCRIPTION
**Pull request by ZilongZhan**
10000

### CODE REVIEW

**Comment by ZilongZhan** [comment]
Accidentally pushed a folder within a folder. Sorry

File: exercise_01/getDistanceFromThreshold.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * output: number
+ */
+
+/**
+ * Find threshold from one number to another
+ * @param {number} value
+ * @param {number} threshold
+ * @returns numerical gap between value and threshold
+ */
+const getDistanceFromThreshold = function (value, threshold) {
```
Comment (tictools) [inline exercise_01/getDistanceFromThreshold.js:14]: This function should return a number but it returns a string... 🤔

FIX: needs work

File: exercise_06/checkCollectionHasElements.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * input: array (collection)
+ * output: boolean
+ */
+
+/**
+ * Check whether a given string is empty or not
+ * @param {array} collection
+ * @returns whether it is true that 'collection' is empty or not
+ */
+function checkCollectionHasElements(collection) {
```
Comment (tictools) [inline exercise_06/checkCollectionHasElements.js:12]: There is an easier way to define this logic.

FIX: needs work

File: exercise_02/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+//import 'getTotalWordsFromString' and execute
+
+import getTotalWordsFromString from "./getTotalWordsFromString.js";
+
+console.log(getTotalWordsFromString("penis")); //5
```
Comment (tictools) [inline exercise_02/index.js:5]: Avoid this kind of words when coding

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by adrianagrd — APPROVED** [review]
Good work! :)

**Review by tictools — APPROVED** [review]

**Pull request merged**